### ISSUE TITLE
New feature request : To import TL431

### ISSUE THREAD

**Issue by nneha12** [sharpie7/circuitjs1#841]
Hi,

Please let me if there is any way to import TL431 in simulation?

**Comment by K0rrid0r** [comment]
Make it into a subcircuit

![image](https://user-images.githubusercontent.com/56724286/215581962-e29880ff-e4cb-4aa7-9a0c-183f483c1b44.png)

**Comment by pfalstad** [comment]
Yes, there's one under Active Building Blocks.  Or hit / and then type TL431, then enter, then drag one out where you want it to go.
